src/components/course/CourseHeader.jsx

```diff
@@ -98,12 +93,44 @@ export default function CourseHeader() {
                     <img
                       src={partner.logoImageUrl}
                       alt={`${partner.name} logo`}
-                      style={{ maxWidth: 160 }}
+                      style={{ maxWidth: 160, maxHeight: 44 }}
                     />
                   </a>
                 ))}
               </div>
             )}
+            <div className={classNames({ 'mb-4': !course.shortDescription })}>
```

Comment:
If it's not too much of an ask, I would love for this course skills to be its own component, it makes it nice and clean and also easier to test. If you do that, you can either (your choice), have a separate test for that component alone or continue using the test you wrote for the course header, since it is embedded in the course header

Reply:
Also, if skillNames comes out empty, with a separate component you can even cleanly not show anything more succinctly as 

```
{course.skillNames.length > 0 && <SkillNames />}
```

```

Reply:
Agree, this needs to be it's own component.

Reply:
Done.